src/qililab/circuit_transpiler/circuit_transpiler.py

```diff
@@ -12,39 +12,42 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""This file contains the functions used to decompose a circuit into native gates and to compute virtual-Z gates."""
-from qibo import gates
-from qibo.models import Circuit
+"""Circuit Transpiler class"""
 
-from qililab.settings.runcard import Runcard
-from qililab.circuit_transpiler.gate_decompositions import translate_gates
-
-from .native_gates import Drag, Wait
-from typing import TYPE_CHECKING
-
-if TYPE_CHECKING:
-    from qililab.platform import Platform, Bus
 
 import contextlib
 from dataclasses import asdict
+from typing import TYPE_CHECKING
 
 import numpy as np
+from qibo import gates
 from qibo.gates import Gate, M
+from qibo.models import Circuit
 
 from qililab.chip.nodes import Coupler, Qubit
+from qililab.circuit_transpiler.gate_decompositions import translate_gates
 from qililab.constants import RUNCARD
-from qililab.instruments import AWG
 from qililab.settings.gate_event_settings import GateEventSettings
 from qililab.typings.enums import Line
 from qililab.utils import Factory
 
 from ..pulse.pulse import Pulse
 from ..pulse.pulse_event import PulseEvent
 from ..pulse.pulse_schedule import PulseSchedule
```

Comment:
When importing things outside of the current submodule (which in this case is `circuit_transpiler`), we should import it from qililab (just to follow the same notation everywhere).

Also, remember that each submodule should contain in its `__init__` all the classes/functions that are used outside of the submodule, such that other files outside that submodule can import all the needed functionality in just one line. In this example:
```suggestion
from qililab.pulse import Pulse, PulseEvent, PulseSchedule
```

Reply:
My second comment above doesn't have to be implemented now in this PR, but I mention it such that everyone in the team has this mentality.

If we follow this, the code will become MUCH clearer and we will stop having circular imports.

Reply:
I will put it in other words: let's try to avoid having two dots in any of the imports, meaning:
```python
from qililab.x.y import z
```
should become:
```python
from qililab.x import z
```